idp/alipay.go

```diff
@@ -204,17 +207,23 @@ func (idp *AlipayIdProvider) postWithBody(body interface{}, targetUrl string) ([
 	if err != nil {
```

Comment:
set this line to `resp, err := idp.Client.Post(targetUrl, "application/x-www-form-urlencoded;charset=utf-8", strings.NewReader(formData.Encode()))` can solve this issue without add extra lib

Reply:
can you help me to test ?

Reply:
@wanghaolonggit have you seen this info , can you help me to test?